.github/workflows/release.yaml

```diff
@@ -29,7 +30,7 @@ jobs:
           export-env: true
         env:
           OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
-          NPM_TOKEN: op://npm-deploy/npm-token/secret
+          NPM_TOKEN: op://npm-deploy/npm-runner-token/secret
```

Comment:
What is this change for?

Reply:
Switches out the token to a longer lived one that only runs in the IP range of the run group